server/client.go

```diff
@@ -91,6 +92,12 @@ type clientOpts struct {
 	Version       string `json:"version"`
 }
 
+// Event data published when a client connects or disconnects
+type clientConnEvent struct {
```

Comment:
Slippery slope, might just be CID and user can collect additional data from monitoring endpoints?